test/e2e/hardware_test.go

```diff
@@ -0,0 +1,599 @@
+// +build all hardware
```

Comment:
Do you think we should name this file more descriptive? I.e. hugepages_test.go?
I am fearful of a catch all bucket of HW tests going into one file.

Reply:
Sure, I will rename this file.